### PR TITLE
chore(one-on-one): add one-on-one meeting template

### LINKED ISSUE #8559: One on one: add 1:1 check-in activity
image

**AC:**

- Create 1:1 category in activity library
- Add check-in meeting there 
- Add check-in meeting to the quick start page
- The check-in has a new `one on one` name, new hover description, new page description (see designs)
- Use a feature flag
- _Think if we need to explicitly mark the meeting as one-on-one. Because on the meeting page we need to show different controls,  or for example in future we want to select all one-on-one check-in meetings_

**Metrics:** 

- one-on-one meeting page seen event
- Modify `Meeting Started` event or create additional event to see that it was not check-in that started but 1:1 meeting

cc @tianrunhe @tghanken 

**[Designs](https://www.figma.com/file/Vc09AD3hxef2ypriihfK0T/Flow---one-on-one?type=design&node-id=0-1&mode=design&t=n98vvQK9920rLKnB-0)**

### PR DESCRIPTION
**Pull request by igorlesnenko**
Fixes #8559

image

image

**How to test:**

- Set `retrosInDisguise` **user** feature flag
- Set `oneOnOne` **org** feature flag to any of your orgs
- Open activity library, see new category and meeting

### CODE REVIEW

File: packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx [hunk 1]
```diff
@@ -205,16 +207,20 @@ const ActivityDetailsSidebar = (props: Props) => {
...
-              const newTeam = availableTeams.find((team) => team.id === teamId)
-              newTeam && setSelectedTeam(newTeam)
-            }}
-            selectedTeamRef={selectedTeam}
-            teamsRef={availableTeams}
-            customPortal={teamScopePopover}
-          />
+          {category === 'oneonone' ? (
+            <>TODO: New user picker will be here</>
+          ) : (
```
Comment (jmtaber129) [inline packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx:212]: +1 Maybe continue using the normal team picker, with a comment TODO to replace it with the user picker?

File: packages/client/components/ActivityLibrary/ActivityDetails/TemplateDetails.tsx [hunk 2]
```diff
@@ -73,6 +73,22 @@ const ACTIVITY_TYPE_DATA_LOOKUP: Record<
...
+const ACTIVITY_ID_DATA_LOOKUP: Record<
+  string,
+  {description: React.ReactNode; integrationsTip: React.ReactNode}
+> = {
+  oneOnOneAction: {
+    description: (
+      <>
+        This is a space to check in one-on-one. Share a personal update using the <b>Icebreaker</b>{' '}
+        phase. Give a brief update on what’s changed with your work during the <b>Solo Updates</b>{' '}
+        phase. Raise issues for discussion in the <b>Agenda</b> phase.
```
Comment (jmtaber129) [inline packages/client/components/ActivityLibrary/ActivityDetails/TemplateDetails.tsx:85]: +1 Is the intention to continue including solo updates in 1:1s? Most 1:1 meetings just follow a single agenda.

File: packages/client/components/ActivityLibrary/ActivityDetails/ActivityDetails.tsx [hunk 3]
```diff
@@ -130,6 +130,7 @@ const ActivityDetails = (props: Props) => {
           teamsRef={teams}
           isOpen={!isEditing}
           type={activity.type}
+          category={category}
```
Comment (jmtaber129) [inline packages/client/components/ActivityLibrary/ActivityDetails/ActivityDetails.tsx:133]: +1 Let's get this from the fragment in the `ActivityDetailsSidebar` component.

**Review by jmtaber129 — CHANGES_REQUESTED** [review]
Looking good! My main concern is adding a completely new category for just one meeting - let's keep this under the "standup" category for now, and either rename that category (e.g. to "check-in" or similar) or move it to its own category sometime in the future.

File: packages/server/postgres/migrations/1690364653869_addOneOnOneMeetingType.ts [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      const hostPath = CDN_BASE_URL.replace(/^\/+/, '')
+      return `https://${hostPath}/store/${partialPath}`
+    }
+    throw new Error('Mssing Env: FILE_STORE_PROVIDER')
+  }
+
+  const oneOnOneActivity = {
+    id: 'oneOnOneAction',
+    type: 'action',
+    name: 'One on One',
```
[the remark this answers is not in the document]
Comment (igorlesnenko) [reply packages/server/postgres/migrations/1690364653869_addOneOnOneMeetingType.ts:25]: I named it according to the designs, so will keep it as is for now, unless there's a strong opinion

File: packages/client/components/ActivityLibrary/ActivityDetails/TemplateDetails.tsx (hunk 2 above)
Comment (igorlesnenko) [reply packages/client/components/ActivityLibrary/ActivityDetails/TemplateDetails.tsx:85]: For now we will have solo updates. Later we will add an option to disable this step

**Comment by igorlesnenko** [comment]
Added `one review required` label as small change under feature flag

**Review by jmtaber129 — APPROVED** [review]

**Pull request merged**